cmd/handler-api.go

```diff
@@ -321,13 +321,21 @@ func maxClients(f http.HandlerFunc) http.HandlerFunc {
 			}
 		}
 
+		globalHTTPStats.addRequestsInQueue(1)
```

Comment:
This makes minio_s3_requests_waiting_total wrong because now this includes requests that are currently processed by the server, was this change only to make the code looks simpler ?